CHANGELOG.md

```diff
@@ -1,5 +1,65 @@
 ### [Versions](https://mui.com/versions/)
 
+## 5.1.1
+
+<!-- generated comparing v5.1.0..master -->
+
+_Nov 15, 2021_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎉 Add `PopperUnstyled` (#29488) @michaldudak
```

Comment:
I actually wouldn't consider it a highlight. The Popper did live in the base package before but caused an issue with imports. I just fixed that.

Reply:
Will remove it.